doc/source/building/build_in_buildservice.rst

```diff
@@ -167,31 +173,60 @@ your dependent packages. These repositories can be provided in two ways:
         </repository>
       </project>
 
+   The above can be simplified further using the path expansion of the last
+   repository to:
+
+   .. code-block:: xml
 
-#. Keep the repositories in your :file:`config.xml` configuration
-   file. If you have installed KIWI as described in
+      <project name="Virtualization:Appliances:Images:openSUSE-Tumbleweed">
+        <title>JeOS for Tumbleweed </title>
+        <description>Host JeOS images for Tumbleweed</description>
+        <repository name="images">
+          <path project="Virtualization:Appliances:Builder" repository="Factory"/>
+          <arch>x86_64</arch>
+        </repository>
+      </project>
+
+   Now `Virtualization:Appliances:Builder` is the last repository, which'
+   repositories are included into the search path. As
+   `openSUSE:Factory/snapshot` is among these, it can be omitted from the
+   repository list.
+
+#. Keep the repositories in your :file:`config.xml` configuration file. If
```

Comment:
Probably i would add  note here with a link to some examples https://build.opensuse.org/project/show/Virtualization:Appliances:Images:Testing_x86 I found them explanatory and there you can also see the _painful_ prjconf configuration.

Reply:
That repository is already mentioned at the bottom of the same file in the section Project configuration. I could add a link to that section?

Reply:
mmm I think this is not the same repository. Note that this is the repository for KIWI testing images, where we do not use `obsrepositories:/` trick so we are capable to build several distro in the same project at a cost of a hard to determine prjconf configuration.

Reply:
Ah, sorry, I was thinking you are referring to a different section. 

So the thing is, I am actually recommending here (as per @mlschroe's recommendation) to keep your normal repositories in `config.xml` and to put the kiwi repository into OBS' config. That will make it impossible to build multiple distros from the same project, too, but it should make working with OBS a lot easier (and it shouldn't require all these nasty `Prefer:`/`Substitute:`/etc. in the project config). Then referring to https://build.opensuse.org/project/show/Virtualization:Appliances:Images:Testing_x86 wouldn't be too helpful, as there these workarounds are unfortunately necessary, as the kiwi repo is in `config.xml` and not in the obs repository configuration.